packages/tools/src/tools/annotation/CobbAngleTool.ts

```diff
@@ -532,6 +579,10 @@ class CobbAngleTool extends AnnotationTool {
       Events.MOUSE_CLICK,
       this._mouseUpCallback as EventListener
     );
+    element.addEventListener(
```

Comment:
it is very odd that we are adding this here, since no other tools has it, but at the same time we don't have any other two step tool. So I don't know

Reply:
I understand. 

Without it, one cannot mouse down+drag to draw the second line segment which is ALSO odd.  If we were to remove it, the unit tests would require some work to fix. Up to you.